keyboards/sofle/info.json

```diff
@@ -0,0 +1,118 @@
+{
+    "keyboard_name": "Sofle",
+    "features": {
+        "swap_hands": true,
```

Comment:
as much as I love the feature, I'm not sure it would be best to enable by default, especially since space is already a constraint here. 

```suggestion
```